### ISSUE TITLE
NTR: CCL4 high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8441]
New term request CCL4 high in blood (elthat_220717052633): C-C motif chemokine ligand 4

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000688
- label: Elevated circulating CCL4 concentration
- parents: [Abnormal circulating CC chemokine concentration (HP:6000446)](https://hpo.jax.org/app/browse/term/HP:6000446)
- pmid: 
- def: The concentration of C-C Motif Ligand 4 (CCL4) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8441
- synonyms:
